tests/integration/test_multi_relations.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+from pathlib import Path
+
+import pytest
+import yaml
+from pytest_operator.plugin import OpsTest
+
+DB_METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+
+
+@pytest.mark.group(1)
+@pytest.mark.abort_on_fail
+async def test_build_and_deploy(ops_test: OpsTest):
+    """Build the charm and deploy 1 units to ensure a cluster is formed."""
+    # Build and deploy charm from local source folder
+    db_charm = await ops_test.build_charm(".")
+
+    config = {"profile": "testing"}
```

Comment:
@taurus-forever

Reply:
Right... I expected to see it as line 34. Missed. Tnx!